cmd/context/utils.go

```diff
@@ -200,7 +203,13 @@ func GetManifest(ctx context.Context, srcFolder string, opts ManifestOptions) (*
 		manifest.Filename = pipelinePath
 		return manifest, nil
 	}
-
+	ctxOptions := &ContextOptions{
+		Show:   true,
+		Silent: opts.Silent,
+	}
+	if err := Run(ctx, ctxOptions); err != nil {
```

Comment:
If I'm not wrong, this would execute the ctx twice for `okteto deploy` and `okteto destroy` command. Moreover, this would be executed whenever we use the GetManifest function. Wouldn't it better to pass the `Silent` option when the context is initialized on each command?